tests/tasks/setup/eks/awscli-cp-with-vpc.yaml

```diff
@@ -81,8 +81,10 @@ spec:
       kubectl apply -k "github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=$(params.aws-ebs-csi-driver-version)"
       # TODO: Calculate replicas based on the cluster size going forward.
       # Patching the coredns not to get scheduled on the monitoring node.
+      # Set revisionHistoryLimit to 0 to avoid keeping older replicasets.
       kubectl patch deployment coredns --patch '{
         "spec": {
+          "revisionHistoryLimit": 0,
```

Comment:
scaling the pods after bringing up the nodes ? Like somewhere here - https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/tests/tasks/setup/eks/awscli-mng.yaml#L117-L123  ?

Reply:
That's another thing that we can try.
Currently we makes all changes to addons before creating nodes.
Let's try this first.